src/utils/css/modules/base-style.scss

```diff
@@ -1,3 +1,5 @@
+@import "./../../../style-config/general";
```

Comment:
this file is not meant to be imported individually, it is imported as part of the `CSS` module here https://github.com/Sage/carbon/blob/v1.5.2/src/utils/css/css.scss which imports all necessary dependencies